src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -105,6 +108,8 @@ public function addEventListener($events, $listener)
 
             if (\is_string($listener)) {
                 unset($this->initialized[$event]);
+            } else {
+                $this->methods[$listener] = $this->getMethod($listener, $event);
```

Comment:
it should be detached in `removeEventListener()` isnt it..

Reply:
To "clean" the $methods content, yes. But is it useful? 🤔

Reply:
i guess not :sweat: registering the same listener again would re-use the same method name still. But it leaks listeners into memory which are not used anymore, so cleaning up sounds nice.

Reply:
Yes I think you are right. Better clean it to not keep references 👍 Thanks for reviewing.